apps/fishing-map-e2e/src/support/app.po.ts

```diff
@@ -36,6 +36,7 @@ export const disablePopups = () => {
   localStorage.setItem('DisableWelcomePopup', 'true')
   localStorage.setItem('DisableSourceSwitchPopup', 'true')
   localStorage.setItem('HighlightPopup', '"vms-with-png"')
+  localStorage.setItem('WelcomePopup', '{"visible":false,"showAgain":false}')
```

Comment:
We have a new popup for the vessel profile and changed the behaviour of the marine manager to follow the same logic.

Reply:
ok good to know, but it is not necessary right know, I will add it when a test in those pages appear